projects/plugins/jetpack/modules/masterbar/admin-menu/admin-menu.css

```diff
@@ -67,6 +67,79 @@
 	padding: 10px 0 10px 8px;
 }
 
+/**
+ * Site Notices
+ */
+#adminmenu a.toplevel_page_site-notices:hover,
+#adminmenu li.toplevel_page_site-notices:hover {
```

Comment:
We can probably apply this to the `:focus` pseudo-selector as well:

<img width="1268" alt="Screen Shot 2021-05-12 at 15 14 00" src="https://user-images.githubusercontent.com/1233880/117980917-bb7af180-b334-11eb-8445-9bf440117611.png">

Reply:
Good point, 8b5baf2.

Reply:
Seems there is another CSS rule with higher priority:

<img width="1275" alt="Screen Shot 2021-05-13 at 15 28 38" src="https://user-images.githubusercontent.com/1233880/118132464-044bac80-b400-11eb-916b-e3344dec5272.png">

Anyway, non-blocking. Can be fixed in a follow-up.

Reply:
Fixed e8b68bc!